### PR TITLE
[release-v1.9] Bump to Go 1.19

### PR DESCRIPTION
**Pull request by pierDipi**
As per title

### CODE REVIEW

**Comment by pierDipi** [comment]
The build image update will only be reflected once merged so we might need to force merge

File: vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml
Comment (creydr) [inline vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml:1]: why was the vendor folder updated in this PR?

**Review by creydr — COMMENTED** [review]
one question. Otherwise lgtm

File: vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml
Comment (pierDipi) [reply vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml:1]: make generate-release did it, I guess 1.8 is wrong as well

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**